UI/org.eclipse.birt.report.designer.ui/src/org/eclipse/birt/report/designer/internal/ui/editors/rulers/EditorRulerLayout.java

```diff
@@ -49,7 +49,7 @@ public Object getConstraint(IFigure child) {
 	 */
 	@Override
 	public void layout(IFigure container) {
-		List children = container.getChildren();
```

Comment:
I suggest using `List<? extends IFigure>` instead, to avoid the explicit cast in line 55.

Reply:
Done.